src/OrchardCore/OrchardCore.DisplayManagement/IDisplayManager.cs

```diff
@@ -6,7 +6,7 @@ namespace OrchardCore.DisplayManagement
     public interface IDisplayManager<TModel>
     {
         Task<IShape> BuildDisplayAsync(TModel model, IUpdateModel updater, string displayType = "", string groupId = "");
-        Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, string groupId = "");
-        Task<IShape> UpdateEditorAsync(TModel model, IUpdateModel updater, string groupId = "");
+        Task<IShape> BuildEditorAsync(TModel model, IUpdateModel updater, bool isNew = true, string groupId = "");
+        Task<IShape> UpdateEditorAsync(TModel model, IUpdateModel updater, bool isNew = true, string groupId = "");
     }
```

Comment:
Here, i would say false for the default value, but i assume that with true you can miss the parameter in more cases.

I didn't try everything but it seems that in some controllers the default value of true is always passed (just seen with intellisense) even we are in an `Edit` method, see the step controller and the queries admin controller.

But because we register instances of `DisplayManager<TModel>` which defines the default value to false, maybe in fact we always pass false. If so, same remark but for the `Create` methods.

Finally, maybe better to not have a default value for this parameter, this to prevent this errors.